src/Logging/Logging/LogMessageFormatter.cs

```diff
@@ -10,9 +10,17 @@ public class LogMessageFormatter
             string timestampFormat)
         {
             var builder = new StringBuilder();
-            builder.Append(FormatTimestamp(message.Timestamp, timestampFormat));
+
+            var timestamp = FormatTimestamp(
+                message.Timestamp,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'message' before using it.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)